src/promptflow/promptflow/executor/_errors.py

```diff
@@ -166,9 +166,23 @@ class LineExecutionTimeoutError(UserErrorException):
     """Exception raised when single line execution timeout"""
 
     def __init__(self, line_number, timeout):
-        self.timeout_sec = timeout
         super().__init__(
-            message=f"Line {line_number} execution timeout for exceeding {timeout} seconds", target=ErrorTarget.EXECUTOR
+            message_format="Line {line_number} execution timeout for exceeding {timeout} seconds",
+            line_number=line_number,
+            timeout=timeout,
+            target=ErrorTarget.EXECUTOR,
+        )
+
+
+class BatchExecutionTimeoutError(UserErrorException):
+    """Exception raised when batch timeout is exceeded"""
+
+    def __init__(self, line_number, timeout):
+        super().__init__(
+            message_format="Processing of line {line_number} exceeded the batch timeout of {timeout} seconds.",
```

Comment:
BatchExecutionTimeout seems not related to line_number?

Reply:
Yes. Just to show which line is affected by the batch timeout when executed.

Reply:
But this sentence might be confusing, looks like one line exceeded 10000 seconds?